.gitignore

```diff
@@ -12,8 +12,8 @@
 node_modules/*
 bower_components/*
 
-# Development user data (settings, database, media, search index, static files)
-development/*
+# Local user data (settings, database, media, search index, static files)
+openslides_data/*
```

Comment:
Both suggestion are ok for me. `local_data` matchs a bit better for `local-installation` option. But `personal_data`explains better that this directory contains *personal* user data. So choose one of them...